WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Summary Section/Edit Order Status/OrderStatusListViewController.swift

```diff
@@ -16,6 +16,11 @@ final class OrderStatusListViewController: UIViewController {
         return ResultsController<StorageOrderStatus>(storageManager: storageManager, matching: predicate, sortedBy: [descriptor])
     }()
 
+    private var orderStatuses: [OrderStatus] {
+        let filteredStatuses = statusResultsController.fetchedObjects.filter { $0.slug != OrderStatusEnum.refunded.rawValue }
```

Comment:
how about filtering the Refunded status in the status results controller level with a predicate, so that we don't need to make other changes in this file?